semantic/carla_controller/labels.py

```diff
@@ -1,27 +1,49 @@
 
 
+# SEMANTIC_CATEGORIES = {
+#     0: "Unlabeled",
+#     1: "Buildings",
+#     2: "Traffic Sign/Lights",
+#     3: "Roads",
+#     4: "Road Lines",
+#     5: "Sidewalk",
+#     6: "Ground",
+#     7: "Vehicles",
+#     8: "Pedestrians",
+# }
+
 SEMANTIC_CATEGORIES = {
     0: "Unlabeled",
-    1: "Buildings",
```

Comment:
Couldn't we have just commented out "Buildings" and achieved the same results?  Or did they need to be offset by 1?

Reply:
Ah, nevermind, you undid all of these label changes in a subsequent PR.